### ISSUE TITLE
Use country selection in Location Content Type

### ISSUE THREAD

**Issue by remdan** [sulu/sulu#6739]
# Problem description

The country is given in the current raw format like the ISO-Code but it's possible to change to everything which can be error-prone.

![Bildschirmfoto 2022-07-30 um 15 38 37](https://user-images.githubusercontent.com/5031692/181916959-bc3b8de6-8360-4f0a-a546-5be94e3a5282.png)

# Proposed solution

Use the same dropdown like in the address form for a contact:

![Bildschirmfoto 2022-07-30 um 15 28 30](https://user-images.githubusercontent.com/5031692/181916850-00d659a7-0dd4-49fc-a1b5-c271a208b4fb.png)

**Comment by alexander-schranz** [comment]
@remdan I agree that would be better but mostly this is more a "Backward compatibility" issue, I think that is why it wasn't changed yet. So the main task here would be how to provide a migration, for custom forms and pages, articles, snippets, ...
